tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/AwsSqsTests.cs

```diff
@@ -119,54 +28,28 @@ public AwsSqsTests(ITestOutputHelper output)
         [SkippableTheory]
         [MemberData(nameof(PackageVersions.AwsSqs), MemberType = typeof(PackageVersions))]
         [Trait("Category", "EndToEnd")]
-        public void SubmitsTraces(string packageVersion)
+        public async Task SubmitsTraces(string packageVersion)
         {
             using var telemetry = this.ConfigureTelemetry();
             using (var agent = EnvironmentHelper.GetMockAgent())
             using (RunSampleAndWaitForExit(agent, packageVersion: packageVersion))
             {
-                var spans = agent.WaitForSpans(_expectedSpans.Count, operationName: "sqs.request");
-                spans.Should().HaveCountGreaterOrEqualTo(_expectedSpans.Count);
-
-                spans.OrderBy(s => s.Start).Should().BeEquivalentTo(_expectedSpans, options => options
-                    .WithStrictOrdering()
-                    .ExcludingMissingMembers()
-                    .ExcludingDefaultSpanProperties()
-                    .AssertMetricsMatchExcludingKeys("_dd.tracer_kr", "_sampling_priority_v1")
-                    .AssertTagsMatchAndSpecifiedTagsPresent("env", "aws.requestId", "aws.queue.url", "runtime-id"));
-                telemetry.AssertIntegrationEnabled(IntegrationId.AwsSqs);
-            }
-        }
+#if NETFRAMEWORK
+                var expectedCount = 56;
+                var frameworkName = "NetFramework";
+#else
+                var expectedCount = 28;
+                var frameworkName = "NetCore";
+#endif
+                var spans = agent.WaitForSpans(expectedCount, operationName: "sqs.request");
```

Comment:
Didn't notice it before, but I have slightly mixed feelings about the `operationName: "sqs.request"` parameter. 

On the one hand, it limits snapshots to only the spans we're really interested in, which is neat. Makes the snapshots simple, and the span count calculations visible.

But my slight concern is it could be hiding "spurious" spans we may be generating? E.g. do we generate HttpClient spans with this integration? Should we? Adding the `operationName` filter means I can't easily answer that question

Reply:
Good point, let me see what additional spans we generate from this. My hypothesis is that it would only be a root-level manual span...let's see!

Reply:
@andrewlock It looks like this adds one manual span (per sync/async routine) and 5 http.request spans (per sync/async routine) because we don't cover all of the SQS client API methods

Reply:
Ultimately, we should add support for these other methods and seeing ALL spans would make it clear that we're not generating extra junk, so I think you're correct that we should compare against all spans